pinecone/sparse/bm25/bm25-quora.ipynb

```diff
@@ -380,10 +380,7 @@
       "source": [
         "import pinecone\n",
         "\n",
-        "pinecone.init(\n",
-        "    api_key=\"YOUR_API_KEY\",  # app.pinecone.io\n",
-        "    environment=\"YOUR_ENV\"  # next to API key in console\n",
-        ")\n",
+        "pinecone.init()\n",
```

Comment:
This will fail for the user, won't it?

I don't see anywhere in the notebook that we instruct the user to get those parameters from pinecone's website and define them as env vars (and what are the env var names).

Also - in the other 3 notebooks, you still ask the user to pass the API_KEY directly as param to `pinecone.init()` in plain text. This has its own disatvanages (what happens if the user shares this notebook with someone?), but at least it's consistent.

Reply:
We should agree on a single method for the user to set their API key, and stick to it throughout all of our notebooks.  
This is true not just for this PR - if you'll go over all of our [example notebooks](https://docs.pinecone.io/docs/examples), they use 3 or 4 different methods for that.

Personally I would prefer a library that asks the user to enter the API-KEY interactively (like asking for a password). I think we have that in some of our notebooks.

Reply:
We decided on using os.env or "API-KEY" solution. I'll talk with James about the other notebooks